order/client.go

```diff
@@ -285,6 +286,51 @@ func (i *Iter) Order() *stripe.Order {
 	return i.Current().(*stripe.Order)
 }
 
+// Return returns all or part of an order.
+// For more details see https://stripe.com/docs/api#return_order.
+func Return(id string, params *stripe.OrderReturnParams) (*stripe.OrderReturn, error) {
+	return getC().Return(id, params)
+}
+
+// Return returns all or part of an order.
+// For more details see https://stripe.com/docs/api#return_order.
+func (c Client) Return(id string, params *stripe.OrderReturnParams) (*stripe.OrderReturn, error) {
+	var body *url.Values
+	var commonParams *stripe.Params
+
+	if params != nil {
+		body = &url.Values{}
+
+		if len(params.Items) > 0 {
+			for _, item := range params.Items {
```

Comment:
Programming in Go: Creating Applications for the 21st Century 
"4.3.4 Key-Ordered Map Iteration"

https://books.google.com/books?id=HW-5SZ1HKusC&pg=PA170&lpg=PA170&dq=%224.3.4+Key-Ordered+Map+Iteration%22&source=bl&ots=Fq903bY-oo&sig=9mfzSgTtYNlPfRoEvNnTuKQDs3o&hl=en&sa=X&ved=0ahUKEwiN_ZKdnenMAhXJWT4KHaZvDv4Q6AEIHTAA#v=onepage&q=%224.3.4%20Key-Ordered%20Map%20Iteration%22&f=false